<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 138, 133);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>5</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Adjust System Time for Touch screen</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>281</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>by KD8CEC</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClose">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>0</y>
       <width>111</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>33</y>
      <width>474</width>
      <height>117</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Adjust time by +, - second</string>
    </property>
    <widget class="QLabel" name="lblDate">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>31</y>
       <width>299</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Date : </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblTime">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>61</y>
       <width>302</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Time : </string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAddSec">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>24</y>
       <width>82</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(245, 121, 0);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>+1
Sec</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSubSec">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>69</y>
       <width>82</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(245, 121, 0);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>-1
Sec</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnZeroMsec">
     <property name="geometry">
      <rect>
       <x>392</x>
       <y>24</y>
       <width>82</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(245, 121, 0);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Set Zero
Milisec</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>153</y>
      <width>475</width>
      <height>164</height>
     </rect>
    </property>
    <property name="title">
     <string>Adjust time by Filds</string>
    </property>
    <widget class="QSpinBox" name="sbMonth">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>46</y>
       <width>118</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox { border: 3px outset grey;padding-right: 40px;} 

QSpinBox::up-button { subcontrol-position: right; right: 40px; width: 40px; height: 35px;}

QSpinBox::down-button { subcontrol-position: right; width: 40px; height: 35px;}</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>12</number>
     </property>
     <property name="value">
      <number>12</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbHour">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>125</y>
       <width>120</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox { border: 3px outset grey;padding-right: 40px;} 

QSpinBox::up-button { subcontrol-position: right; right: 40px; width: 40px; height: 35px;}

QSpinBox::down-button { subcontrol-position: right; width: 40px; height: 35px;}</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>24</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbSec">
     <property name="geometry">
      <rect>
       <x>251</x>
       <y>124</y>
       <width>142</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox { border: 3px outset grey;padding-right: 40px;} 

QSpinBox::up-button { subcontrol-position: right; right: 40px; width: 40px; height: 35px;}

QSpinBox::down-button { subcontrol-position: right; width: 40px; height: 35px;}</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>59</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbMinute">
     <property name="geometry">
      <rect>
       <x>128</x>
       <y>125</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox { border: 3px outset grey;padding-right: 40px;} 

QSpinBox::up-button { subcontrol-position: right; right: 40px; width: 40px; height: 35px;}

QSpinBox::down-button { subcontrol-position: right; width: 40px; height: 35px;}</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>59</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbDay">
     <property name="geometry">
      <rect>
       <x>127</x>
       <y>46</y>
       <width>122</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox { border: 3px outset grey;padding-right: 40px;} 

QSpinBox::up-button { subcontrol-position: right; right: 40px; width: 40px; height: 35px;}

QSpinBox::down-button { subcontrol-position: right; width: 40px; height: 35px;}</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>31</number>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>27</y>
       <width>388</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 87, 83);
color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>1</y>
        <width>67</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Year</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>2</y>
        <width>67</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Month</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>2</y>
        <width>67</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Day</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>90</y>
       <width>388</width>
       <height>36</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 87, 83);
color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>18</y>
        <width>37</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Hour</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>18</y>
        <width>67</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Min</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>18</y>
        <width>50</width>
        <height>15</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sec</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClose_2">
      <property name="geometry">
       <rect>
        <x>38</x>
        <y>1</y>
        <width>79</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Zero</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClose_3">
      <property name="geometry">
       <rect>
        <x>163</x>
        <y>0</y>
        <width>79</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Zero</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClose_4">
      <property name="geometry">
       <rect>
        <x>307</x>
        <y>0</y>
        <width>79</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Zero</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>394</x>
       <y>27</y>
       <width>80</width>
       <height>136</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(136, 138, 133);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btnGetDateTime">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>79</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>Get 
DateTime</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSetDateTime">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>79</width>
        <height>74</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(245, 121, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Set
Date Time</string>
      </property>
     </widget>
    </widget>
    <widget class="QSpinBox" name="sbYear">
     <property name="geometry">
      <rect>
       <x>251</x>
       <y>46</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox { border: 3px outset grey;padding-right: 40px;} 

QSpinBox::up-button { subcontrol-position: right; right: 40px; width: 40px; height: 35px;}

QSpinBox::down-button { subcontrol-position: right; width: 40px; height: 35px;}</string>
     </property>
     <property name="minimum">
      <number>1900</number>
     </property>
     <property name="maximum">
      <number>3000</number>
     </property>
     <property name="value">
      <number>1900</number>
     </property>
    </widget>
    <zorder>sbMonth</zorder>
    <zorder>sbHour</zorder>
    <zorder>sbSec</zorder>
    <zorder>sbMinute</zorder>
    <zorder>sbDay</zorder>
    <zorder>widget_2</zorder>
    <zorder>frame_2</zorder>
    <zorder>sbYear</zorder>
    <zorder>widget</zorder>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
